Exiled.API/Features/Player.cs

```diff
@@ -1505,6 +1505,26 @@ public static bool AddToWhitelist(string userId, bool isPermanent)
         /// </summary>
         public static void ReloadWhitelist() => WhiteList.Reload();
 
+        /// <summary>
+        /// Plays a gun sound at the specified position.
+        /// </summary>
+        /// <param name="position">Position to play the sound at.</param>
+        /// <param name="firearmType">The type of firearm to play the sound of.</param>
+        /// <param name="maxDistance">The maximum distance the sound can be heard from.</param>
+        /// <param name="audioClipId">The audio clip ID to play.</param>
+        public static void PlayGunSound(Vector3 position, ItemType firearmType, byte maxDistance = 45, byte audioClipId = 0)
```

Comment:
since this is not for a specific player and seems global it should be in Map class